### DISCUSSION TITLE
Can't visit the resulting url afte pushing my code and show "Task timed out after 20.81 seconds" error !

### DISCUSSION THREAD

**Discussion by chenalex207** [deta/space-docs#159] [Q&A]
Dear all,
             I am writing a Node.js site for doing some simple things. The local development is fine after using "space dev" and it look good. But after I push my code using "space push", the resulting url it returned can not visit and return error message like below:     Have anyone seen this before ? I have been working that for few days. Can anyone have good suggestion for that ?  Thanks very much ...

Error Type:
Error Message:
2023-07-31T04:55:02.303Z c424be4e-f501-44dd-bd87-6b94604fb1c7 Task timed out after 20.81 seconds

Logs:
time="2023-07-31T04:54:31Z" level=info msg="serving logs listener on sandbox.localdomain:1234" agent=logsApiAgent
TELEMETRY	Name: telemetry-extension	State: Subscribed	Types: [Function]
App listening on port 4200!
time="2023-07-31T04:54:41Z" level=info msg="serving logs listener on sandbox.localdomain:1234" agent=logsApiAgent
TELEMETRY	Name: telemetry-extension	State: Already subscribed	Types: [Function]
App listening on port 4200!

**Comment by lemonyte** [comment]
That error message means your serverless function is taking longer than 20 seconds to respond. The issue in your case is that your web server is running on port 4200 instead of getting the port from the $PORT variable.
